src/data_enrichment/paraphraser.py

```diff
@@ -0,0 +1,35 @@
+from transformers import AutoTokenizer, AutoModelForSeq2SeqLM
+
+class Paraphraser:
+    def __init__(self):
+        self.model_name = "humarin/chatgpt_paraphraser_on_T5_base"
```

Comment:
make this a parameter in config file, we shouldn't ideally hard code this @pamodaDilranga

Reply:
Created config files and updated the code